content/rs/references/crdb-cli-reference.md

```diff
@@ -17,7 +17,14 @@ Each instance is responsible for updating the instances residing on other partic
 Write conflicts are resolved using [CRDTs]({{< relref "/rs/administering/active-active.md" >}}).
 
 To programmatically maintain an Active-Active database and its instances, you can use the CRDB-CLI command-line tool.
-With CRDB-CLI you can:
+
+## Using the crdb-cli
+
+To use the CRDB-CLI tool, you must log in to the terminal of an RS host with a user that belongs to the group that RS was installed with (Default: **redislabs**).
```

Comment:
To use the CRDB-CLI tool, use SSH to log in to a RS host with a user belonging to the group that RS was installed with (Default: **redislabs**).